### DISCUSSION TITLE
Dynamic staking TOTAL EARNED

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#17] [Bugs and Reports]
![image](https://github.com/user-attachments/assets/877b83ab-7812-4f10-86d0-bc323a0b7c9d)
Total earned shows last cycle and not grand total like in Conservative view. I think it should be changed to grand total of all dynamic stakes

**Comment by bf-monster** [comment]
Hey, we will discuss this.thank you

**Comment by bf-mike-mkt** [comment]
can we reopen this topic pls?

**Reply by bf-monster** [reply]
Fixed
